### PR TITLE
Put both players' life totals on a single shared debouncer.

### PR DESCRIPTION
**Pull request by vslashg**
For slicker Seige Rhinos and Lightning Helices!

### CODE REVIEW

File: gamestate.js [hunk 1]
```diff
@@ -123,28 +123,23 @@ function updateValue(elt, newValue, toggleOnZero) {
...
-var updateP2Numbers = _.debounce(function(v) {
-    updateValue($(".p2.life"), v.val().life);
-    updateValue($(".p2.poison"), v.val().poison, true);
-    if (v.val().gamewins > 0) {
-        updateValue($(".p2.wins"), v.val().gamewins);
-    } else {
-        $(".p2.wins").text(v.val().gamewins);
-    }
+var updateAllNumbers = _.debounce(function () {
+    firebase.database().ref('player1').once('value', updateSide.bind(null, ".p1"));
```
Comment (kprevas) [inline gamestate.js]: Does this work?  It seems like it would wait for another update.

Comment (vslashg) [inline gamestate.js]: I tested this, and it works.  (Live at http://verylowsodium.com/urza/overlay-game.html).  What a _debounce instance does, when called, is save the arguments (none here of course), and start a timer.  If it's called again before the timer expires, the timer is restarted and the new arguments are saved.

When the timer finally expires, the wrapped object is called with the latest saved arguments.

So in this case, when things have stopped changing after 1000ms, we fetch the current values of 'player1' and 'player2' and do all updates.

Comment (kprevas) [inline gamestate.js]: I was concerned that the call to once() inside the debounced function would wait for another database update.  It doesn't though?

Comment (vslashg) [inline gamestate.js]: Aha, I misunderstood the part that threw you.  Each call to .on() or .once() creates a new listener that will trigger right away (as soon as the DB is ready).  once('value', fn) is thus an idiom for doing a single read.

See https://firebase.google.com/docs/reference/js/firebase.database.Reference#on

Comment (kprevas) [inline gamestate.js]: Oh nice, I was only looking at the docs for once() which don't mention that.  That's super handy.

**Pull request merged**